.github/workflows/ci-test-jit.yaml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_call:
 
 env:
-  runtime_tests_version: "@unison/runtime-tests/releases/0.0.2"
+  runtime_tests_version: "@unison/runtime-tests/releases/0.0.1"
```

Comment:
Is this the switch back to v4?

Reply:
Yeah, the change to 0.0.2 was just to point it at different files.